.github/workflows/pr-review-advisor-repair.yaml

```
@@ -0,0 +1,575 @@
+      contents: read
+      pull-requests: read
+    outputs:
+      artifact_ids: ${{ steps.metadata.outputs.artifact_ids }}
+      attempt_key: ${{ steps.select.outputs.attempt_key }}
+      base_sha: ${{ steps.select.outputs.base_sha }}
+      head_ref: ${{ steps.select.outputs.head_ref }}
+      selected: ${{ steps.select.outputs.selected }}
+      selection_artifact_id: ${{ steps.upload-selection.outputs.artifact-id }}
+      source_head_sha: ${{ steps.select.outputs.source_head_sha }}
+    env:
+      AUDIT_OUTPUT_DIR: ${{ runner.temp }}/phase1-audit
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)